src/_extensions/acceptance-criteria.md

```diff
@@ -9,20 +9,38 @@ license: cc-by-3.0
 ## Contents
 
 - [Introduction](#intro)
+- [One goal policy](#one-goal-policy)
 - [Extension Metadata](#metadata)
 - [Content](#content)
 - [Development](#development)
 - [Monetization and data processing](#monetization)
 
 ## Introduction {#intro}
 
-Your extension must satisfy all the following acceptance criteria to be included in Opera Addons.
+Your extension must satisfy all the following acceptance criteria to be included in Opera Addons. When these criteria get more specific or restricted, authors of currently published extensions are required to upload updated packages.
+
+All researchers and Opera users are invited to report improper extensions by using the "Reported issues" functionality on respective addon pages.
+
+## One goal policy {#one-goal-policy}
+
+Your extension needs to have one, clearly stated purpose. Proper examples include:
+- display a weather forecast after clicking a button
```

Comment:
Other lists have capital letters and full stops so these two lists should probably also have that.